information-system/data-store/access-control.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: default
+title: Access Control
+nav_order: 9
+grand_parent: Information System
+parent: Data store
+---
+
+{: .no_toc }
+
+<details  open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+{: .text-delta }
+* TOC
+{:toc}
+____
+</details>
+
+# Access Control Overview
+
+This document will discuss the types of access controls available and then reveal the methods to apply these access controls through the Datastore User Interface. 
+
+## What access controls are available for datastore datasets?
+ 
+There are three access rights to consider: 
+
+1. Metadata access
+2. Data access
+3. Admin access
+
+
+## Metadata access
+
+Facilitates control of general viewing and metadata updating.
+
+### Metadata Read
+
+- If enabled, users with this right will be able view the dataset's metadata and find the dataset in the datastore "search" and "list" tools. 
+- If disabled, users will not be able to find the dataset in the datastore let alone view the metadata.
+
+
+### Metadata Write
+
+- If enabled, users can update metadata fields and will be granted metadata read access regardless of the setting for metadata read.
+- If disabled, general users will not be able to update the metadata.
+
+## Data access
+
+Facilitates control over the upload/download of data to/from the storage location associated with the dataset.
+
+### Data read
+
+- If enabled, the download panel will become visible to users and they will be able to download data from the storage location. 
+- If disabled, the download panel will not be visible and users will not be able to download data from the storage location.
+
+### Data write
+
+- If enabled, users will be able to upload/overwrite and download to/from the storage location as both the upload and download panels will be made available to general users.
+- If disabled, users will not be able to upload/overwrite data in the storage location.
+
+## Admin access
+
+- If enabled, provides administrative control over the dataset. That is, the ability to lock and unlock the dataset in addition to all of the above privileges. For more information on locking and unlocking, see here.
```

Comment:
I would like "lock and unlock" directly to the locking a dataset page rather than explicitly calling it out. 

Also worth mentioning that admin control provides access control i.e. 

## Admin access

If granted, provides full control over a dataset. Administrators of a dataset are granted all of the above permissions, and can additionally: 

- reconfigure access (see below)
- apply or remove _dataset locks_ (linked)

Reply:
Resolved